luaui/Widgets/gui_game_type_info.lua

```diff
@@ -44,22 +42,47 @@ function widget:ViewResize()
 	widgetScale = (0.80 + (vsx * vsy / 6000000))
 
 	font = WG['fonts'].getFont(nil, 1.5, 0.25, 1.25)
+
+	if( messages[1] ) then
```

Comment:
Just a minor note, parenthesis here is not needed.

Reply:
Force of habit from a C++ dev. Is there a style guide somewhere for the lua widgets?

Reply:
Not that I know of, try to follow what me, Floris or Sprung have written :D